### ISSUE TITLE
In rare cases the "add new email-adress" email is sent repeatedly instead of the verification email on registration

### ISSUE THREAD

**Issue by sblindow** [liqd/a4-meinberlin#5330]
**URL:**
---
**user:**
non-verified user account

**expected behaviour:**
upon creating a new user account, a verification email is sent out to the user, so they can confirm their email-adress

**behaviour:** 
As described by a user:

"Wie ich schon schrieb, habe ich meine E-Mail-Adresse bereits mehrfach bestätigt (durch Klicken auf den Link in der Bestätigungs-E-Mail). Nach der ersten Bestätigung meiner E-Mail-Adresse wurde ich bei der darauffolgenden Anmeldung aufgefordert, meine E-Mail-Adresse meinem Account auf mein.berlin.de <http://mein.berlin.de/> „hinzuzufügen“. Zwar habe ich mich gewundert, warum ich eine bereits „bestätigte" E-Mail-Adresse nun außerdem noch „hinzufügen“ soll, habe aber wie gefordert, den Link zu dieser Hinzufügung angeklickt. Beim nächsten Anmeldeversuch wurde ich erneut aufgefordert, meine E-Mail-Adresse durch Klick auf den Link zu „bestätigen". Also habe ich meine E-Mail-Adresse zum zweiten Mal bestätigt. Daraufhin habe ich das Browserfenster geschlossen, um mich neu in einem neuen Fenster auf mein.berlin.de <http://mein.berlin.de/> anzumelden. Nun wurde ich erneut aufgefordert, meine E-Mail-Adresse meinem Account „hinzuzufügen“. Nach dieser zweifachen Wiederholung und insgesamt vier vergeblichen Anmeldeversuchen bin ich zu dem Schluss gekommen, dass ich in einer Schleife gelandet bin und dass meine Registrierung in dem System nicht vervollständigt wird. "

**important screensize:**
---
**device & browser:**
If it is relevant, we can contact the user.

**Comment/Question:** 

Screenshot?

**Comment by goapunk** [comment]
reading the text again it kind of sounds like the user only clicked on the link in the email, not on the button to confirm the email on the page. And if you then sign-in instead, you are asked to confirm your email again and get a different email asking if you want to add that email to your account. @sblindow do you remember if they ever mentioned clicking on the actual button and not just the link in the email?

**Comment by sblindow** [comment]
@goapunk Yes! This must have been what happened. Although we didn't have any further correspondece besides the quoted email so we can't confirm 100%. I also noticed, that the second email, asking you if you want to add your adress to your account only gets sent after a few minutes. When I tried to log in without clicking the button immediately after creating the account, only the text referencing the confimation e-mail was shown, withou the blue banner, indicating that an email has been sent.

**Comment by CarolingerSeilchenspringer** [comment]
@sblindow will close for now due to dependencies we cannot change.
